package_manager/dpkg.bzl

```diff
@@ -6,7 +6,8 @@ exports_files(deb_files + ["packages.bzl"])
 """)
 
     args = [
-        repository_ctx.path(repository_ctx.attr._dpkg_parser),
+        # dpkg_parser.par must be built separately beforehand (bazel build ... //package_manager:dpkg_parser.par)
+        str(repository_ctx.path(Label("//:WORKSPACE")).dirname) + "/bazel-bin/package_manager/dpkg_parser.par",
```

Comment:
This is a problem as these `bazel-*` directories are a convenience and may not be present (e.g., readonly file system or [`--symlink_prefix`](https://docs.bazel.build/versions/master/user-manual.html#flag--symlink_prefix)).  Let me ask on the Bazel slack for ideas.

Reply:
Thanks. All I need is to be able to reference a binary. We can assume `--symlink_prefix` or anything affecting the path is not set.

Reply:
@briandealwis have you had a chance to get some feedback from a Bazel expert?

Reply:
@nlopezgi agreed that they've seen other rules embed build directory layout like this and there are many cases in which there is no way around it. We could put some strings (such as `bazel-bin`) in top-level variables and add checks for directory names in a follow-up as suggested.